lib/mrtrix3/app.py

```diff
@@ -1101,6 +1101,88 @@ def _is_option_group(self, group):
            not group == self._positionals and \
            group.title not in ( 'options', 'optional arguments' )
 
+  class TypeBoolean:
+    def __call__(self, input_value):
+      true_var = "True"
+      false_var = "False"
+      converted_value = None
+      if input_value.lower() == true_var.lower():
+        converted_value = True
+      elif input_value.lower() == false_var.lower():
+        converted_value = False
+      else:
+        raise argparse.ArgumentTypeError('Entered value is not of type boolean')
+      return converted_value
```

Comment:
For consistency with C++ command-line behaviour, see https://github.com/MRtrix3/mrtrix3/blob/3.0.4/core/mrtrix.h#L285-L293.

Reply:
Changes available in commit: https://github.com/ankitasanil/mrtrix3/pull/1/commits/8d91a38c27bb2340584b4c3ecb5d3c9556dd6096